andela/ah-maps
# [Feature 161255240] Add email confirmation after registration

#### What does this PR do?
 This PR adds email confirmation after registration.

#### Description of Task to be completed?
 Have the following endpoint working:
  `GET /user/activate/<str:token>`

#### How should this be manually tested?
- After cloning the repo, cd into it and run: `python manage.py runserver`
- Register a user and check registered email's inbox
- Click the link in the email to complete registration

#### What are the relevant pivotal tracker stories?
#161255240

#### Screenshots
<img width="1277" alt="screen shot 2018-11-01 at 13 17 59" src="https://user-images.githubusercontent.com/37045046/47846368-aecac980-ddd8-11e8-9de4-b18bbb4aec0e.png">
